src/components/LeafletMap.js

```diff
@@ -319,6 +357,7 @@ class LeafletMap extends Component {
           this.removeHighlight();
           this.setIconActive(markerPoint);
           siteClickHandler(marker);
+          sitesDropDownToggle(false);
```

Comment:
I don't believe you should be executing the same method that produces the same effect multiple times when doing it once should suffice

Reply:
You are correct. Retracting...